README.md

```diff
@@ -7,59 +7,75 @@ in a uniform way.
 Installation
 ------------
 Serviced can run on a single host or across multiple hosts. First, serviced
-depends on docker.
+depends on docker. The minimal version of docker should be **19.03.11** for running serviced **1.8.x**.
+The minimum supported versions of Linux distributions are Ubuntu is 18.04 and CentOS 7.6.
 
-1. Follow the instruction at http://www.docker.io/ , to install
+1.  Follow the instruction at http://www.docker.io/ , to install
    it on every host that serviced will run on. Ensure docker is running.
 
-2. Install a generated package
+2.  Install a generated package
+      ```bash 
+          sudo apt-key adv --keyserver keys.gnupg.net --recv-keys AA5A1AD7
+    ￼     REPO=http://get.zenoss.io/apt/ubuntu
+          sudo sh -c 'echo "deb [ arch=amd64 ] '${REPO}' trusty universe" > /etc/apt/sources.list.d/zenoss.list'
```

Comment:
There's an odd character on line 19.

Since `REPO` is used only once, I suggest replacing the `${REPO}` with `http://get.zenoss.io/apt/ubuntu` and remove the `REPO` variable entirely.

This shell snippet should be copy/paste-able to the command line and removing that variable would enable copy/pasting.